src/main/java/pl/sda/carrentalmanager/services/CarService.java

```diff
@@ -1,4 +1,25 @@
 package pl.sda.carrentalmanager.services;
 
+import org.springframework.stereotype.Service;
+import pl.sda.carrentalmanager.models.Car;
+import pl.sda.carrentalmanager.repositories.CarRepository;
+
+import java.util.List;
+
+@Service
 public class CarService {
+
+    private final CarRepository carRepository;
+
+
+    public CarService(CarRepository carRepository) {
+        this.carRepository = carRepository;
+    }
+
+    public List<Car> getAll() {
```

Comment:
Dobrą praktyką jest trzymanie jednakowych nazw, czyli jeśli w repository jest metoda `findAll()` to w serwisie tez powinna sie tak nazywac